assets/php/examples/master_detail/project_list.php

```diff
@@ -49,33 +52,38 @@ protected function Form_Create() {
 			// now is time to add some trick to filter only by some field
 			 
 			// remove filter from ID column
-			$colId = $this->dtgProjects->AddConnectedColumn('Id');
+			$colId = new QDataGridColumn('Id', '<?= $_ITEM->Id ?>');
 			$colId->FilterType = "";
+			$this->dtgProjects->AddColumn($colId);
 
-			$this->dtgProjects->AddConnectedColumn('Name', 'Name=Project');
-			$this->dtgProjects->AddDbTypeColumn('ProjectStatusTypeId', 'ProjectStatusType');
+			$this->dtgProjects->AddColumn(new QDataGridColumn('Project', '<?= $_ITEM->Name ?>'));
+			$this->MetaAddTypeColumn(QQN::Project()->ProjectStatusTypeId, 'ProjectStatusType');
```

Comment:
The same reason here - I don't know how this whole example can be implemented in the QDataGrid2 and if it is possible in the first place. Moreover, if QDataGrid is deprecated there are many places where it is used in examples. Should we replace it in all places?